webapp-django/crashstats/crashstats/static/crashstats/js/socorro/dashboard_graph.js

```diff
@@ -91,12 +91,9 @@ $(function() {
         return ajaxURL;
     },
     buildCrashReports = function(data) {
-        var useTmpl = data.date_range_type === "build" ? crashReportsByBuildDateTmpl : crashReportsByVersionTmpl,
-            crashReportsHTML = Mustache.to_html(useTmpl, data),
-            releaseChannelsContainer = $("#release_channels");
-
-        releaseChannelsContainer.empty().append(crashReportsHTML);
-        releaseChannelsContainer.find('h4').each(function(index, item) {
+        $("#release_channels")
+        .find('h4')
```

Comment:
Not a criticism but why do you do that? I've seen it before. Why not `$('#release_channels h4').each(...`?